.circleci/config.yml

```diff
@@ -112,7 +112,7 @@ jobs:
           paths:
             - geth/<<pipeline.parameters.system-contracts-path>>
 
-  unit-tests:
+  race:
```

Comment:
I think you forget to add -race?

Reply:
You might want to just consolidate `unit-tests` and `coverage`, and leave `race` as it is?

Reply:
So I guess I dropped the race flag by accident when I standardised the approach to running tests, probably copy and pasted the block from unit-tests to race.